stdlib/importlib/metadata/__init__.pyi

```diff
@@ -61,7 +61,56 @@ class EntryPoint(_EntryPointBase):
         def attr(self) -> str: ...
     if sys.version_info >= (3, 10):
         dist: ClassVar[Distribution | None]
-        def matches(self, **params: Any) -> bool: ...  # undocumented
+        def matches(
```

Comment:
By the way, I don't really like the fact that I manually copy-and-pasted the typings of all methods which are based on `matches()`. I thought about maybe using a `TypedDict` for defining a general interface for `params`. Is there some convention for how to do this?

Reply:
> Is there some convention for what to do in such cases?

No, the way you've done it is the best way possible at the moment -- but there are discussions on the typing-sig mailing list right now about a possible PEP proposing being able to use a `TypedDict` for kwargs in a function signature :)

Reply:
Cool :) I also feel like a solution in the fashion of `ParamSpec` could be interesting (that is, some way to type-annotate that a set of keyword arguments are literally forwarded as-is to some other method).